docker/Makefile

```diff
@@ -14,7 +14,15 @@ image: prepare
 	docker build -f Dockerfile -t $(IMAGE_NAME) $(BUILD_DIR)
 
 run: image
-	docker-compose -f docker-compose.yml up
+    # This will not work on Windows
```

Comment:
Is this OK?

Reply:
Yeah, we don't officially support Windows dev environments, so this is totally fine.